frontend/src/components/Wrapper.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <h2><slot name="title"></slot></h2>
```

Comment:
```suggestion
    <h2>{{title}}</h2>
```

Reply:
here is how we can use title

Reply:
hey, I tried it, but I need to use props, is it better to use props or slots in this situation?